examples/playground-movies/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "playground-movies-example",
```

Comment:
https://github.com/gentrace/gentrace-node/blob/f2adddcad40b13d1b92b90408f5e244d87290f6a/examples/files/package.json#L2

We normally follow the convention `@examples/playground-movies`